README.md

```diff
@@ -105,18 +176,31 @@ go build
 ./dexc --testnet
 ```
 
-This will open up a terminal-based interface for the client.
+Connect to the client from your browser at `localhost:5758`.
+
+## Contribute
 
-## Development
+**Looking to contribute? We need your help** to make DEX &#35;1.
 
-Before starting work, chat with us in the
+Nearly all development is done in Golang and JavaScript. Work is coordinated
```

Comment:
Would prefer to go with the proper name "Go" https://tip.golang.org/doc/faq#go_or_golang